.changesets/fix_garypen_3249_docker_heaptrack.md

```diff
@@ -0,0 +1,9 @@
+### set DEBUG_IMAGE=true when building debug docker image ([Issue #3249](https://github.com/apollographql/router/issues/3249))
```

Comment:
I think it would be more valuable to make this (not just here, but in all changesets) a user-impacting title statement whenever possible.  Otherwise, I don't know if this title means enough to the user as it stands.  It also makes it very digestible if we remove the bodies and just keep the titles.

```suggestion
### Restore missing debug tools in "debug" Docker images ([Issue #3249](https://github.com/apollographql/router/issues/3249))
```